submissions/medinatapanpa/javascript/js/african.js

```diff
@@ -0,0 +1,34 @@
+let african = [
+    {
+        name:'Basil/scentleaf',
```

Comment:
Well done. It seems african.js and nigerian.js do the same thing. Can you remove one?

Reply:
They do the same thing, but different methods, what to be sure both methods are acceptable.